use_case_examples/cifar/cifar_brevitas_finetuning/CifarInFhe.ipynb

```diff
@@ -62,7 +63,10 @@
     "bit = 5\n",
     "seed = 42\n",
     "\n",
-    "device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
+    "# To use the CUDA backend for the Concrete compiler, install the GPU-supported version.\n",
+    "# Refer to the official guide:\n",
+    "# https://github.com/zama-ai/concrete-ml/blob/main/docs/guides/using_gpu.md\n",
+    "device = \"cuda\" if check_gpu_available() else \"cpu\"\n",
```

Comment:
the PyTorch `device` and CML `device` may differ. You should use the original

`device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",` 

for pytorch training, and a separate device variable for CML compilation (based on the CML check for gpu availability)